aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -189,14 +189,19 @@ def _expand_dims_to_match_ome(
         # need to correct channel count if this is a RGB image
         n_samples = ome.images[scene_index].pixels.channels[0].samples_per_pixel
         for d in dims:
+            # SizeC can represent RGB (Samples) data rather
+            # than channel data, whether or not this is the case depends
+            # on what the SamplesPerPixel are for the channel
```

Comment:
is there a scenario where you can have N channels and M samples and SizeC is set to N*M?  Do you have to do a bit of math to find the true N value? Or is that automatically handled here.  I remember that coming up a long time ago as a possibility.

Reply:
Ah the channel dimension shape is determined by line 196 rather than line 200 so instead of using `SizeC` it uses the length of the channels list, likely already for this reason.